src/models/Order.js

```diff
@@ -69,10 +72,14 @@ const orderSchema = new mongoose.Schema(
 // Create a function to update order statuses
 const updateOrderStatus = async () => {
   try {
-    // Get the current date and subtract 1 minute
-    const oneMinutesAgo = new Date(Date.now() - 1 * 60 * 1000);
+    let isDevelopment = process.env.NODE_ENV !== 'production';
+    const timeDuration = isDevelopment
+      ? parseInt(process.env.DEV_TIME_DURATION)
+      : parseInt(process.env.PROD_TIME_DURATION);
```

Comment:
Maybe give these a clear name so that if someone is looking at just the `.env` file it's clear what these are used for. Eg: `DEV_ORDER_EXPIRY_DURATION` or something

Reply:
@akosasante sure! thank you